README.md

```diff
@@ -172,15 +179,15 @@ All the following people who have contributed to the extension:
 * Johannes Rössel
 * Joshua Stafman
 * Kevin Palmowski
-* Loïc PÉRON
+* Loïc PÉRON
```

Comment:
The characters with diacritics in this line were changed from single characters to [composite characters](https://en.wikipedia.org/wiki/Precomposed_character):
- `ï` (U+00EF) became `ï` (U+0069, U+0308)
- `É` (U+00C9) became `É` (U+0045, U+0301)

I assume this change was unintended, so it should probably be reversed.

Reply:
If correctly understand this change the chars have been changed to the UTF-8 encoding, should be fine

Reply:
Hmm, I'm not sure that's the case. From what I could ascertain, `ï` (U+00EF) is encoded in UTF-8 as the byte sequence `C3 AF`, whereas the composite `i`+`¨` is `69 CC 88`; and similarly, `É` (U+00C9), is encoded in UTF-8 as `C3 89`, whereas the composite `E`+`´` is encoded as `45 CC 81`.

The new part of the diff uses the composed characters, so something other than encoding conversion seems to have caused the issue.

Reply:
I've opened #259 to restore the original non-composed characters in this line.